pkg/controller/usersignup/usersignup_controller.go

```diff
@@ -533,8 +533,14 @@ func (r *ReconcileUserSignup) DeleteMasterUserRecord(mur *toolchainv1alpha1.Mast
 func (r *ReconcileUserSignup) sendDeactivatedNotification(logger logr.Logger, userSignup *toolchainv1alpha1.UserSignup) error {
 	notification := &toolchainv1alpha1.Notification{
 		ObjectMeta: v1.ObjectMeta{
-			Name:      userSignup.Status.CompliantUsername + "-deactivated",
+			GenerateName:      userSignup.Status.CompliantUsername + "-" + toolchainv1alpha1.NotificationTypeDeactivated + "-",
```

Comment:
```suggestion
			GenerateName:      fmt.Stringf("%s-%s-", userSignup.Status.CompliantUsername, toolchainv1alpha1.NotificationTypeDeactivated,
```

Reply:
Fixed as part of: 21c323901080723156e76ab6563614f16880baf3